app/services/dttp/retrieve_providers.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Dttp
+  class RetrieveProviders
+    include ServicePattern
+
+    class HttpError < StandardError; end
```

Comment:
can we get to a take a pick https://github.com/DFE-Digital/register-trainee-teachers/blob/1288198b921a94671f0fc17fcb1faa22ebe4ed9d/app/services/dttp/retrieve_users.rb#L7

Reply:
Changing it to `Error` as the user retreival stuff has already been merged.
However, I think this might be one to be picked up as tech debt for [consistency across the codebase](https://github.com/DFE-Digital/register-trainee-teachers/search?q=HttpError)
And since not all errors might be `HttpError`s, maybe this will need a bit of thought.